packages/site/src/blocks/ImageBlock.tsx

```diff
@@ -0,0 +1,3 @@
+export const ImageBlock = () => {
+    return null;
```

Comment:
Could we add generate-block-types in this Package as well ? (I'm not sure if this is easily achievable, but if so, it would be great.

https://github.com/vivid-planet/comet/blob/main/packages/site/cms-site/package.json#L19

Reply:
This will be done in a follow-up PR